### PR TITLE
Lesson 4.5

### PR DESCRIPTION
**Pull request by pstnv**
Lesson 4.5 CSS Layout:

1) added Experience CSS;
2) added variables
      2.1) for most recurring colors;
      2.2) for automatic calculation (button size and position);
      2.3) for header (to avoid overlapping content).
3) made sticky header.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -15,8 +24,11 @@ body {
 }
 
 *::selection {
-	background-color: #828c7f;
-	background-color: rgba(196, 226, 226, 1.00);
+	background-color: var(--blue-color);
```
Comment (dargaCode) [inline css/index.css:27]: great use of variables!

File: css/index.css [hunk 2]
```diff
@@ -233,52 +259,85 @@ header nav a:not(:first-child):hover {
     padding: 15px;
     background-color: #ffffff;
     font-size: 16px;
+	font-family: 'Preahvihear', sans-serif;
     filter: drop-shadow(2px 2px 2px rgba(0, 0, 0, 0.5));
     transition: background-color .4s linear;
 }
 
 .skills_list li:nth-child(3n+1) {
```
Comment (dargaCode) [inline css/index.css:267]: These left color borders look really nice =D

File: css/index.css [hunk 3]
```diff
@@ -233,52 +259,85 @@ header nav a:not(:first-child):hover {
...
+	justify-content: center;
+}
+
+.experience_list li {
+	margin: 15px;
+	padding: 15px;
+	min-width: 110px;
+	border: 1px solid #ffffff;
+	font-size: 16px;
+	font-family: 'Preahvihear', sans-serif;
```
Comment (dargaCode) [inline css/index.css:326]: You write this same line inside of a lot of different selectors. Should it actually be applied to the whole document? Or maybe you were having an issue with browser defaults overriding these fonts somehow?

**Review by dargaCode — COMMENTED** [review]
Excellenttttttt

File: css/index.css (hunk 3 above)
Comment (pstnv) [reply css/index.css:326]: > You write this same line inside of a lot of different selectors. Should it actually be applied to the whole document? Or maybe you were having an issue with browser defaults overriding these fonts somehow?

Yes, when I put to body tag = "one font" and to all headers = "another font" after some time while I scrolling page headers fonts are overriding with fonts from body tag.

**Pull request merged**